.travis.yml

```diff
@@ -4,11 +4,11 @@ python:
   - "2.7"
   - "3.6"
   - "3.7"
-  - "3.8-dev"
+  - "3.8"
 
 matrix:
     allow_failures:
-        - python: "3.8-dev"
+        - python: "3.8"
```

Comment:
I think we should remove this exception entirely. 3.8 should not be allowed to fail